spec/display_board_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'stringio'
+
+def display_board(console, board)
+    $stdout = console
```

Comment:
This produces an undesired side effect of changing the value of `$stdout`.  Try calling `console.puts` instead of plain `puts` on line 6, below, to avoid this.